spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1064,15 +1080,6 @@
                 reason: 'Provided secondary proof of address',
               ),
             )
-
-            expect(job_analytics).to have_logged_event(
-              'GetUspsProofingResultsJob: Success or failure email initiated',
-              email_type: 'Failed',
-              enrollment_code: pending_enrollment.enrollment_code,
-              service_provider: anything,
-              timestamp: anything,
-              wait_until: nil,
-            )
```

Comment:
this expectation was redundant with the `'enrollment_with_a_status_update'` shared example above